PSRT/Public/Find-RTTicket.ps1

```diff
@@ -0,0 +1,83 @@
+Function Find-RTTicket {
+    <#
+    .SYNOPSIS
+        Find RT tickets with a specified query
+    .DESCRIPTION
+        Find RT tickets with a specified query
+    .PARAMETER Query
+        Query string to pass to RT
+
+        Details: https://rt-wiki.bestpractical.com/wiki/REST#Ticket_Search
+    .PARAMETER Expand
+        If specified, pull ticket information for each ticket returned by this query
+    .PARAMETER Referer
+        Referer to use for whitelisting purposes.  See @ReferrerWhitelist RT configuration
+        
+        Defaults to 'https://<hostname>', which translates to <hostname>:443 on RT's side
+    .PARAMETER Session
+        RT session to use.  Defaults to PSRTConfig.Session (Created by New-RTSession)
+    .PARAMETER BaseUri
+        Base URI for RT.  Defaults to PSRTConfig.BaseUri
+    .PARAMETER Raw
+        If specified, do not parse output
+    .EXAMPLE
+        Find-RTTicket -Query "Created > '3 days ago'"
+    .EXAMPLE
+        Find-RTTicket -Query "Owner='wframe' AND Status='new'" -Referer http://WhiteListedHostname
+    .EXAMPLE
+        Find-RTTicket -Query "Status='new'" -Expand
+    .FUNCTIONALITY
+        Request Tracker
+    #>
+    [cmdletbinding()]
+    Param(
+        [parameter(Position = 1)]
+        [string]$Query,
+
+        [string]$Referer = "https://$ENV:COMPUTERNAME",
+
+        [switch]$Expand,
+
+        [Parameter( ValueFromPipeLine = $true, 
+            ValueFromPipelineByPropertyName = $true )] 
+        [ValidateNotNull()] 
+        [Microsoft.PowerShell.Commands.WebRequestSession] 
+        $Session = $PSRTConfig.Session,
+        [ValidateNotNull()] 
+        [string]$BaseUri = $PSRTConfig.BaseUri,
+        [switch]$Raw
+    )
+    $InvokeParams = @{ WebSession = $Session }
+    if($Referer)
+    {
+        $headers = @{}
+        $headers.Add('Referer', $Referer)
+        $InvokeParams.Add('Headers', $headers)
+    }
+    $Query = [System.Web.HttpUtility]::UrlEncode($Query)
+    $uri = Join-Parts -Separator '/' -Parts $BaseUri, "/REST/1.0/search/ticket?query=$Query"
+
+    $Response = ( Invoke-WebRequest @InvokeParams -Uri $uri).Content
+    if ($Raw) {
+        $Response
+    }
+    else {
+        $Tickets = ConvertFrom-RTResponse -Content $Response
+        if($Expand)
+        {
+            'Raw', 'Query', 'Referer', 'Expand'| Foreach { [void]$PSBoundParameters.Remove($_) }
```

Comment:
We use splatting when calling `Get-RTTicket`.  We take the bound parameters hashtable that PowerShell creates automatically, remove the parameters that `Get-RTTicket` doesn't support (e.g. `Query`) on this line, and then splat the remaining bits.  Example outcome:

*User specifies a session*:  Session is in PSBoundParameters, passed on to Get-RTTicket
*User does not specify a session*:  Session is not in PSBoundParameters, so Get-RTTicket uses the same default as Find-RTTicket (`$PSRTConfig.Session`)

Same deal with BaseUri

This is just in case someone doesn't want to use the BaseUri/Session from the config